assets/js/controllers/settings/sweepAll.controller.js

```diff
@@ -0,0 +1,33 @@
+angular
+  .module('walletApp')
+  .controller('SweepAllController', SweepAllController);
+
+function SweepAllController($scope, $q, $timeout, $translate, Wallet, Alerts) {
+  $scope.alerts = [];
+  $scope.defaultIdx = Wallet.my.wallet.hdwallet.defaultAccountIndex;
+  $scope.addresses = Wallet.legacyAddresses().filter(a => a.active && !a.isWatchOnly);
```

Comment:
It might be helpful to display these addresses and their amounts in the first step of the Sweep All modal. Something like: 
![screen shot 2016-04-01 at 11 35 55 am](https://cloud.githubusercontent.com/assets/5640772/14211787/ecf4c86e-f7fd-11e5-9035-56db3914c639.png)